backup.sh

```diff
@@ -9,11 +9,18 @@
 echo "Backup started at $(date "+%Y-%m-%d %H:%M:%S")"
 
 CURRENT_DATE=$(date +%Y%m%d%H%M)
+
+# Intermediate working backup directory
+WORKDIR="/backup/$CURRENT_DATE"
```

Comment:
This removed one intentional change we had. 

By re-using `/backup/`, also the intermediate file are created in whatever is mapped as the `/backup/` volume.

We intentionally used `/backup_tmp/`. Background is, that this enables mapping `/backup/` to any kind of (remote) share (S3, CIFS, SFTP, SSHFS, ...) but only write the final result there. 

One possible workaround would be to make `WORKDIR` and `TARGETDIR` settable from outside via an Environment Variable. This would allow to have a default which would still work without problems with local volumes but allows to split them apart in remote volume scenarios.

Reply:
@pfeigl Yes that's correct. I open a new Issue for this case. #36

Reply:
@pfeigl I want to separate pushing to remote directory (also required additional volume mapping) from the simple backup process. Please lock at this PR #38 . Your feedback is very welcome.